msvc/sdk-build.targets

```diff
@@ -38,7 +38,7 @@
       <GetPackageContentsDependsOn>$(GetPackageContentsDependsOn);$(ComputeLinkInputsTargets)</GetPackageContentsDependsOn>
   </PropertyGroup>
 
-  <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents"  DependsOnTargets="$(GetPackageContentsDependsOn)" Condition="'$(ConfigurationType)' == 'DynamicLibrary' or '$(ConfigurationType)' == 'Application'">
+  <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents" DependsOnTargets="$(GetPackageContentsDependsOn);_GenerateProjectPriFileCore" Condition="'$(ConfigurationType)' == 'DynamicLibrary' or '$(ConfigurationType)' == 'Application'">
```

Comment:
1) example?

2) where is that?

---
In reply to: [108993817](https://github.com/Microsoft/WinObjC/pull/2372#discussion_r108993817) [](ancestors = 108993817)